crates/sui-graphql-rpc/src/types/date_time.rs

```diff
@@ -25,23 +24,15 @@ impl ScalarType for DateTime {
     }
 
     fn to_value(&self) -> Value {
-        Value::String(format!("{}", self))
+        Value::String(format!("{:?}", self.0))
```

Comment:
nit: Could you leave a comment here explaining the expected format of `chrono::DateTime`'s `Debug` format?